src/components/WordGroupSelector/WordGroupSelector.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import { collectedWordGroupsService } from "util/GWUtil/resource";
+import "./WordGroupSelector.css";
+import WordGroupIcon from "./WordGroupIcon";
+import WordSelector from "./WordSelector";
+
+function WordGroupSelector(props) {
+  var collectedWordGroups = null;
+  var wordKeys = null;
+  const [wordGroups, setWordGroups] = useState({});
+
+  useEffect(() => {
+    collectedWordGroupsService.all().then(function(res) {
+      collectedWordGroups = res;
```

Comment:
you can just do `collectedWordGroupsService.all().then(function(collectedWordGroups) {` instead of renaming the `res` variable